### PR TITLE
Fix bug where meetings are not filtered

### PR DESCRIPTION
**Pull request by sebastiantoh**
Think some conflicts weren't resolved properly resulting in this bug. 

P.S. I noticed this for sales as well, I'm not sure if this is intended although it seems to filter sales correctly from my preliminary testing of `sale list`

Changes:
- Resolve bug with meetings not being filtered
- Update consistency of Edit Command messages (add part specifying that contact indices must be positive)
- Fix faulty examples
- Updated sample data for meetings (@AaronnSeah I renamed them so that it doesn't look too repetitive)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -454,7 +456,7 @@ Format: `reminder edit INDEX [c/CONTACT_INDEX] [m/MESSAGE] [d/DATETIME] [st/STAT
 
 Examples:
 * `reminder edit 1 c/2` edits the 1st reminder to be associated with the second contact in the displayed contact list.
-* `reminder edit 3 m/Follow up call d/2020-11-28 13:00` edits the message and scheduled date of the 3rd reminder to be "Follow up call" and "28th November 2020, 1PM" respectively.
+* `reminder edit 3 m/ d/2020-11-28 13:00` edits the message and scheduled date of the 3rd reminder to be "Call to follow up" and "28th November 2020, 1PM" respectively.
```
Comment (hakujitsu) [inline docs/UserGuide.md]: Should there be a message specified behind `m/`?

**Review by hakujitsu — CHANGES_REQUESTED** [review]
Besides the one comment, LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (sebastiantoh) [reply docs/UserGuide.md]: Thanks for pointing that out!

**Review by hakujitsu — APPROVED** [review]
LGTM, merging now!

**Pull request merged**